### PR TITLE
Defaults the gym video recorder fps to match episode decimation

### LINKED ISSUE #892: [Bug Report] The length of the video from gym.wrappers.RecordVideo does not match the episode length
The length of the video produced by gym.wrappers.RecordVideo (e.g., in the workflow examples) does not always match the episode length. 

This is because the RecordVideo wrapper uses "render_fps" metadata, which is, by default, 30. If the simulation frame rate divided by decimation is not exactly 30, the length of the produced video will be different from the episode length (i.e., the environment inside the video will play faster or slower than the intended speed). 

Since an important use case for Isaac Lab is sim2real applications, having a video that is at the correct length/fps with the real counterpart is handy. While this metadata could easily be set by the workflow scripts a posterior, I don't see a reason why such shouldn't be the default behavior.

I will go ahead and create a PR for this, as it has a minor fix that does not seem to break anything.

### System Info

- Commit: df184e1
- Isaac Sim Version: 4.1
- OS: Ubuntu 22.04
- GPU: RTX 4090 
- CUDA: 12.2
- GPU Driver: 535.129.03

### Checklist

- [x] I have checked that there is no similar issue in the repo (**required**)
- [x] I have checked that the issue is not in running Isaac Sim itself and is related to the repo

### PR DESCRIPTION
**Pull request by ozhanozen**
Sets the "render_fps" metadata used by the gym.wrappers.RecordVideo such that, if it is used, the produced video will have the correct episode length (i.e., the environment in the video will play at the correct speed). 

The correct render_fps should be equal to `1 / (self.cfg.decimation * self.cfg.sim.dt)` which I integrated to the init function of the ManagerBasedRLEnv. This seems to work well.

Fixes #892

## Type of change

- Bug fix (non-breaking change which fixes an issue)

## Screenshots

Screenshot 2024-08-28 at 18 10 30

## Checklist

- [x] I have run the [`pre-commit` checks](https://pre-commit.com/) with `./isaaclab.sh --format`
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have updated the changelog and the corresponding version in the extension's `config/extension.toml` file
- [x] I have added my name to the `CONTRIBUTORS.md` or my name already exists there

### CODE REVIEW

File: source/extensions/omni.isaac.lab/omni/isaac/lab/envs/manager_based_rl_env.py [hunk 1]
```diff
@@ -81,6 +81,8 @@ def __init__(self, cfg: ManagerBasedRLEnvCfg, render_mode: str | None = None, **
         self.common_step_counter = 0
         # -- init buffers
         self.episode_length_buf = torch.zeros(self.num_envs, device=self.device, dtype=torch.long)
+        # -- set the framerate of the gym video recorder wrapper so that the duration of the produced video matches the episode length
+        self.metadata["render_fps"] = 1 / (self.cfg.decimation * self.cfg.sim.dt)
```
Comment (Mayankm96) [inline source/extensions/omni.isaac.lab/omni/isaac/lab/envs/manager_based_rl_env.py]: Should this be int?

```suggestion
        self.metadata["render_fps"] = int(1 / (self.cfg.decimation * self.cfg.sim.dt))

```

**Review by Mayankm96 — COMMENTED** [review]
Could you please also add it here?

> https://github.com/isaac-sim/IsaacLab/blob/main/source/extensions/omni.isaac.lab/omni/isaac/lab/envs/direct_rl_env.py#L186-L188

File: source/extensions/omni.isaac.lab/omni/isaac/lab/envs/manager_based_rl_env.py (hunk 1 above)
Comment (ozhanozen) [reply source/extensions/omni.isaac.lab/omni/isaac/lab/envs/manager_based_rl_env.py]: No, I think it shouldn't be.

I initially thought the same but gym.wrappers.RecordVideo does work with floating input. If the 1/dT is a floating number for some reason, and you set render_fps to be int, then we would have the same problem of mismatching fps.

**Comment by ozhanozen** [comment]
> Thanks a lot for the fix! Could you please also add it here?
> 
> > https://github.com/isaac-sim/IsaacLab/blob/main/source/extensions/omni.isaac.lab/omni/isaac/lab/envs/direct_rl_env.py#L186-L188

I have added. 

Also, I have changed the code comment slightly to this:
`# -- set the framerate of the gym video recorder wrapper so that the playback speed of the produced video matches the simulation`

File: source/extensions/omni.isaac.lab/omni/isaac/lab/envs/manager_based_rl_env.py [hunk 2]
```diff
@@ -81,6 +81,8 @@ def __init__(self, cfg: ManagerBasedRLEnvCfg, render_mode: str | None = None, **
         self.common_step_counter = 0
         # -- init buffers
         self.episode_length_buf = torch.zeros(self.num_envs, device=self.device, dtype=torch.long)
+        # -- set the framerate of the gym video recorder wrapper so that the playback speed of the produced video matches the simulation
+        self.metadata["render_fps"] = 1 / (self.cfg.decimation * self.cfg.sim.dt)
```
Comment (Mayankm96) [inline source/extensions/omni.isaac.lab/omni/isaac/lab/envs/manager_based_rl_env.py]:
```suggestion
        self.metadata["render_fps"] = 1 / self.step_dt

```

File: source/extensions/omni.isaac.lab/omni/isaac/lab/envs/direct_rl_env.py [hunk 3]
```diff
@@ -184,6 +184,9 @@ def __init__(self, cfg: DirectRLEnvCfg, render_mode: str | None = None, **kwargs
             if "startup" in self.event_manager.available_modes:
                 self.event_manager.apply(mode="startup")
 
+        # -- set the framerate of the gym video recorder wrapper so that the playback speed of the produced video matches the simulation
+        self.metadata["render_fps"] = 1 / (self.cfg.decimation * self.cfg.sim.dt)
```
Comment (Mayankm96) [inline source/extensions/omni.isaac.lab/omni/isaac/lab/envs/direct_rl_env.py]:
```suggestion
        self.metadata["render_fps"] = 1 / self.step_dt
```

**Review by Mayankm96 — APPROVED** [review]

**Pull request merged**